packages/cli/test/fixtures/relation/index.js

```diff
@@ -83,12 +83,24 @@ const SourceEntries = {
     content: readSourceFile('./repositories/order-class-type.repository.ts'),
   },
 
-  NoKeyModel: {
+  NokeyModel: {
```

Comment:
I think this should stay as `NoKeyModel`

Reply:
I want to make it consistent as the model name is `Nokey` in `nokey.model.ts`.

Reply:
Should we update the one in `nokey.model.ts` then?

Reply:
either is fine..